brainstorm/src/BrainstormModel.ts

```diff
@@ -103,17 +115,23 @@ export function createBrainstormModel(fluid: FluidContainer): BrainstormModel {
 
     SetNoteColor,
 
+    // Set the note as liked or unliked by the author
```

Comment:
```suggestion
    // Set or unset the note as liked by the user
```
`author` was not the best variable name here :/

Reply:
hmm but the only person who can trigger this method is the author 🤔

Reply:
wait really?  you can't like other people's notes?

Reply:
my bad, I was thinking something else. Yes, `user` would be a better name here, and you can like other people's not since 
 `noteId` is an argument.